src/main/java/net/mcreator/ui/dialogs/wysiwyg/InputSlotDialog.java

```diff
@@ -43,7 +43,7 @@ public class InputSlotDialog extends AbstractWYSIWYGDialog<InputSlot> {
 	public InputSlotDialog(WYSIWYGEditor editor, @Nullable InputSlot slot) {
 		super(editor, slot);
 		setModal(true);
-		setSize(850, 435);
+		setSize(850, 470);

```

Comment:
Try matching width so borders align:

![image](https://github.com/MCreator/MCreator/assets/16374228/503c0bdf-f316-4dfe-890c-e609752940a1)

![image](https://github.com/MCreator/MCreator/assets/16374228/a68a3faf-034f-4050-8f2f-87c41b35a23d)

Same with output dialog.

Reply:
This just makes it fir with current width, I would prefer solution where you just change window width to align them in the default state as it is less changes